lib/keys.dart

```diff
@@ -17,4 +17,15 @@ class Keys {
   static const inboxListTiles = Key('inboxListTiles');
   static const starEmailButton = Key('star_email_button');
   static const replyExit = Key('ReplyExit');
+  static const shrineUsernameField = Key('shrineUsernameField');
+  static const shrinePasswordField = Key('shrinePasswordField');
+  static const shrineAppTab = Key('shrineAppTab');
+  static const shrineContextMenu = Key('shrineContextMenu');
+  static const productItem = Key('productItem');
```

Comment:
This one as well has weird name, it should at least indicate that it is a key to some text